recipe/meta.yaml

```diff
@@ -22,13 +22,13 @@ requirements:
         - python  # [win]
         - cmake  # [win]
         - msinttypes  # [win and py<35]
-        - jpeg
-        - zlib
         - bison  # [unix]
         - flex  # [unix]
+        - zlib
+        - jpeg 9b
```

Comment:
At least we can grep the feedstocks repo and know what needs to be re-built in case of emergency :smile: